test/unit/annotation_spec.js

```diff
@@ -2281,7 +2302,10 @@ describe("annotation", function () {
         { ref: buttonWidgetRef, data: buttonWidgetDict },
       ]);
       const task = new WorkerTask("test print");
-      partialEvaluator.options = { ignoreErrors: true };
+      partialEvaluator.options = {
+        ignoreErrors: true,
+        standardFontDataUrl: null,
```

Comment:
Since `standardFontDataUrl: null,` is the default value, is this line necessary?